Formula/k/klee.rb

```diff
@@ -105,13 +104,15 @@ def install
     # https://github.com/klee/klee/blob/v#{version}/README-CMake.md
     args = %W[
       -DKLEE_RUNTIME_BUILD_TYPE=Release
+      -DENABLE_KLEE_LIBCXX=ON
       -DKLEE_LIBCXX_DIR=#{libcxx_install_dir}
-      -DKLEE_LIBCXX_INCLUDE_DIR=#{libcxx_install_dir}/include/c++/v1
+      -DKLEE_LIBCXX_BC_PATH=#{libcxx_install_dir}/lib
+      -DKLEE_LIBCXX_INCLUDE_DIR=#{libcxx_install_dir}/include
+      -DKLEE_LIBCXX_INCLUDE_PATH=#{libcxx_install_dir}/include/c++/v1
```

Comment:
It looks like upstream removed some cache variables in https://github.com/klee/klee/commit/f27b748ba2be259177aa672e4d5fc39d36479c7f#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aL509

However, based on documentation, it doesn't seem that was intended. So may need to report upstream.

As workaround, `find_file`/`find_path` creates cache entries so I added the `KLEE_LIBCXX_BC_PATH` and `KLEE_LIBCXX_INCLUDE_PATH` to bypass this issue.